docs/advanced-guide/serving-static-files/page.md

```diff
@@ -0,0 +1,76 @@
+# Serving Static Files using GoFr
+
+Often times we require to serve static content be it a default profile image or a static website. We want to have a mechanism to serve those content without having a hassel of implementing it from scratch.
```

Comment:
`Often, we require` missing comma + often instead of often times

`static content be it a default` => `static content like a default` 

`Those contentS` plural

Reply:
Thanks for the remarks @ccoVeille , will update as per the recommendations.